.github/release.yml

```diff
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+      - deploy
+      - infrastructure
```

Comment:
Can we add documentation on these tags to the contributor guide?

Reply:
yep, will be updating the docs after this PR goes in